### PR TITLE
Musixmatch frontend

### PR DESCRIPTION
**Pull request by seunomonije**
Untested front-end code from yesterday. Thought y'all should take a look.

### CODE REVIEW

File: client/js/analysis.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+window.onload = function() {
+    // Listen for on submit click
+    let formSubmit = document.getElementById("formSubmit");
+    formSubmit.addEventListener("click", fetchAndShowResponse, false);r
```
Comment (laptou) [inline client/js/analysis.js:5]: What's this `r`?

File: client/js/analysis.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (laptou) [inline client/js/analysis.js:1]: This file should be indented with 2 spaces

File: client/analysis.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (laptou) [inline client/analysis.html:1]: This file should be indented with 2 spaces

File: client/analysis.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
+
+
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="STEP Capstone">
+        <script src="js/analysis.js"></script>
```
Comment (laptou) [inline client/analysis.html:9]: Consider moving the `script` tag to the end of your `body`. This way, your tag will have access to the DOM when it loads, instead of having to listen for a "page loaded" event.

Comment (seunomonije) [reply client/analysis.html:9]: will do.

**Comment by jessicaslaughter** [comment]
You should get Grace's approval before requesting from me and Nick, just so the three of y'all get a lot of chances to review code before seeing our input. Feel free to ping anyone if you haven't gotten a review in 24 hours!

**Comment by seunomonije** [comment]
I'm gonna try and merge this anyways because we can use it with Youtube instead

File: client/analysis.html (hunk 4 above)
Comment (jessicaslaughter) [reply client/analysis.html:9]: Looks like this still needs to be resolved?

File: client/js/analysis.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    changeInnerText("response", value);
+    return value;
+}
+
+/**
+ * Change the contents of div
+ * @param elId the id of the element 
+ * @param value the value to insert as innerText
+ */
+function changeInnerText(elId, value) {
```
Comment (jessicaslaughter) [inline client/js/analysis.js:24]: Consider renaming this "setInnerText", and updating the function comment to be more generic since the element isn't necessarily a div (this helper could later be used for something other than the "response" div).

File: client/analysis.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
+
+
```
Comment (geekster777) [inline client/analysis.html:2]: Remove unnecessary whitespace

File: client/analysis.html [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+        <meta name="description" content="STEP Capstone">
+        <script src="js/analysis.js"></script>
+        <link rel="stylesheet" href="css/analysis.css">
+        <title>Song Analysis</title>
+    </head>
+
+    <body>
+        <h1>Enter a song name and artist name</h1>
+
+        <p>Insert Song name</p>
```
Comment (geekster777) [inline client/analysis.html:17]: You should use `<label>`'s for the text describing an input.

File: client/js/analysis.js (hunk 2 above)
Comment (geekster777) [reply client/js/analysis.js:1]: This was never completed

File: client/js/analysis.js (hunk 5 above)
Comment (geekster777) [reply client/js/analysis.js:24]: +1. I'd also consider getting rid of this helper entirely since it makes the parent function less easy to read. In general, you probs don't need a helper function if it's under 3 lines of code (unless they're complicated lines of code).

File: client/js/analysis.js (hunk 2 above)
Comment (geekster777) [reply client/js/analysis.js:1]: Please actually complete this. This file doesn't use 2-space indents and it should

File: client/analysis.html (hunk 3 above)
Comment (geekster777) [reply client/analysis.html:1]: Please use 2 space indents

File: client/js/analysis.js (hunk 2 above)
Comment (seunomonije) [reply client/js/analysis.js:1]: Ah my fault...I mixed up 2-space indents and 2 lines at the top of the file. My bad.

Comment (geekster777) [reply client/js/analysis.js:1]: Please remove the 2 lines at the top of the file as well :)
